assets/js/base/context/hooks/cart/use-store-cart.ts

```diff
@@ -85,7 +82,7 @@ const defaultCartTotals: CartResponseTotals = {
 const decodeValues = (
 	object: Record< string, unknown >
 ): Record< string, unknown > =>
-	fromEntriesPolyfill(
+	Object.fromEntries(
```

Comment:
I know you have a separate PR for this, but I think it's a good change now 👍🏻  Very little share for ie11 and we don't test it anyway.